proposals/0002-root-signature-in-clang.md

```diff
@@ -385,13 +384,13 @@ parsedRootSignature = RootSignature{
 };
 ```
 
-### Root Signatures in the LLVM IR
+### LLVM IR Root Signature Representation
 
-During frontend code generation an IR-based representation of the root signature
-is generated from the in-memory data structures stored in the AST. This is
-stored as metadata nodes, identified by named metadata. The metadata format
-itself is a straightforward transcription of the in-memory data structure - so
-it is a list of root elements.
+LLVM IR code generation, during entry point emition,
+an IR-based representation of the root signature is generated from the in-memory 
+data structures stored in the AST. This is stored as metadata nodes, identified 
+by named metadata. The metadata format itself is a straightforward transcription 
+of the in-memory data structure - so it is a list of root elements.
```

Comment:
I don't understand why this has been changed.  Before reviewing it too closely (I'm having trouble understanding the new version) I think it'd help to understand what different information is intended to be communicated.